client/tests/nft_test.rs

```diff
@@ -40,78 +46,65 @@ async fn test_get_nft_address_by_index() -> anyhow::Result<()> {
     Ok(())
 }
 
-// ---------------------nft get item metadata tests
-
+// ---- Tests methods return valid data.
 #[tokio::test]
-async fn test_get_nft_content_uri() -> anyhow::Result<()> {
+async fn test_get_nft_data_is_valid() -> anyhow::Result<()> {
     common::init_logging();
     let client = common::new_mainnet_client().await;
     let factory = TonContractFactory::builder(&client).build().await?;
     let contract = factory.get_contract(&assert_ok!(
         "EQCGZEZZcYO9DK877fJSIEpYMSvfui7zmTXGhq0yq1Ce1Mb6".parse()
     ));
     let res = assert_ok!(contract.get_nft_data().await);
-    assert_eq!(
-        res.individual_content,
-        MetaDataContent::External {
-            uri: "https://nft.fragment.com/number/88805397120.json".to_string()
-        }
-    );
-    let meta_loader = assert_ok!(NftItemMetaLoader::default());
-    let content_res = assert_ok!(meta_loader.load(&res.individual_content).await);
-    assert_eq!(
-        content_res.name.as_ref().unwrap(),
-        &String::from("+888 0539 7120")
-    );
-    assert_eq!(
-        content_res.image.as_ref().unwrap(),
-        &String::from("https://nft.fragment.com/number/88805397120.webp")
-    );
-    Ok(())
-}
 
-#[tokio::test]
-async fn test_get_nft_content_arkenston() -> anyhow::Result<()> {
-    common::init_logging();
-    let client = common::new_mainnet_client().await;
-    let factory = assert_ok!(TonContractFactory::builder(&client).build().await);
-    let contract = factory.get_contract(&assert_ok!(
-        "EQDhR36C8pSVtyhOFtE9nh2DFq4WYUbTZFmvjfnShlrXq2cz".parse()
+    let expected_collection_address = assert_ok!(TonAddress::from_base64_url(
```

Comment:
please check this construction with clippy. no need to convert `&'static str` to String and then back to `&str`. Also method TonAddress::parse()  might be handy

Reply:
Updated. I forgot to run cargo clippy with the --test flag.